src/parquet/thrift/CMakeLists.txt

```diff
@@ -22,13 +22,29 @@ set(THRIFT_SRCS
 set_source_files_properties(parquet_types.cpp PROPERTIES
   COMPILE_FLAGS -Wno-unused-variable)
 
-add_library(parquet_thrift STATIC
-  ${THRIFT_SRCS}
-)
+if (PARQUET_BUILD_SHARED)
+  add_library(parquet_thrift_shared SHARED
+    ${THRIFT_SRCS}
+  )
```

Comment:
Ok, but consider the fact that I removed several times my BUILD_DIRECTORY so all CMakeCache files should be removed.